pkg/gitreceive/k8s_util.go

```diff
@@ -15,10 +15,10 @@ const (
 	slugBuilderName   = "deis-slugbuilder"
 	dockerBuilderName = "deis-dockerbuilder"
 
-	tarURLKey        = "TAR_URL"
-	putURLKey        = "put_url"
+	tarPath          = "TAR_PATH"
+	putPath          = "PUT_PATH"
```

Comment:
It seems like the slugbuilder will start looking for this new env var after https://github.com/deis/slugbuilder/pull/41. That's been marked on this PR as required for merge